neutron_vpnaas/services/vpn/device_drivers/ovn_ipsec.py

```diff
@@ -212,7 +219,7 @@ def find_vpn_port(self, ptype, router_id, host):
                           ptype=ptype, router_id=router_id, host=host)
 
 
-class OvnSwanDriver(ipsec.IPsecDriver):
+class OvnSwanDriver(meter_ipsec.MeterIPsecDriver, ipsec.IPsecDriver):
```

Comment:
maybe inherit Meter here is not good.
How about self.meterMgr = meter_ipsec.MeterManager()
then call self.meterMgr.xxx()